### DISCUSSION TITLE
Fix permissions for comskip

### DISCUSSION THREAD

**Discussion by jzhvymetal** [OpenSageTV/Community#50] [Docker]
I redeploved my SageTV docker on UnRaid.  I notice that comskip would not work.   The permissions for the comskip files need to be fixed in the docker.  I ran the following to in a console to correct the permissions:

chmod 777 /opt/sagetv/comskip/*

**Comment by jusjoken** [comment]
Can you let me know which docker container you used?

Thanks

**Reply by jzhvymetal** [reply]
sagetvopen-sagetv-server-java11
